mne/bem.py

```diff
@@ -1048,19 +1048,17 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
     os.makedirs(op.join(ws_dir, 'ws'))
     run_subprocess(cmd, env=env, stdout=sys.stdout, stderr=sys.stderr)
     #
-    os.chdir(ws_dir)
     if op.isfile(T1_mgz):
         # XXX : do this with python code
-        surfaces = [subject + '_brain_surface', subject +
-                    '_inner_skull_surface', subject + '_outer_skull_surface',
-                    subject + '_outer_skin_surface']
-        for s in surfaces:
+        surfs = ['brain_surface', 'inner_skull', 'outer_skull', 'outer_skin']
+        for s in surfs:
+            s = op.join(ws_dir, '%s_%s_surface' % (subject, s))
```

Comment:
Yes, GitHub changed the way emails work if you open the link it takes you to the old page, look at the top there is a "You are viewing a subset of the changes. View all changes"